background.js

```diff
@@ -2,16 +2,28 @@
 
 chrome.webRequest.onBeforeRequest.addListener(function (details) {
 	if (details.method === 'GET') {
+		const isOpenQuery = 'q=is:open';
+		const pjaxQueryStart = details.url.search(/\?_pjax=%23js-repo-pjax-container/);
+
+		let newUrl;
+		if (pjaxQueryStart !== -1) {
+			newUrl = details.url.substr(0, pjaxQueryStart + 1) + isOpenQuery;
```

Comment:
```suggestion
			newUrl = details.url.slice(0, pjaxQueryStart + 1) + isOpenQuery;
```

I prefer `.slice` for consistency.